ts/modules/data.src.ts

```diff
@@ -258,15 +258,15 @@ const {
 
 import '../parts/Chart.js';
 import '../mixins/ajax.js';
+import H from '../parts/Globals.js';
 
 // Utilities
 var addEvent = Highcharts.addEvent,
     Chart = Highcharts.Chart,
     win = Highcharts.win,
     doc = win.document,
     merge = Highcharts.merge,
-    fireEvent = Highcharts.fireEvent,
-    SeriesBuilder: typeof Highcharts.SeriesBuilder;
+    fireEvent = Highcharts.fireEvent;
```

Comment:
Utilities function should be used via import. Also import statements should be sorted, so that named imports come first and anonymous come last.